vnext/Shared/Modules/WebSocketModule.cpp

```diff
@@ -119,9 +146,18 @@ GetOrCreateWebSocket(int64_t id, string &&url, weak_ptr<WebSocketModule::SharedS
 
 namespace Microsoft::React {
 
+#pragma region WebSocketModule
+
 WebSocketModule::WebSocketModule() : m_sharedState{std::make_shared<SharedState>()} {
   m_sharedState->ResourceFactory = [](string &&url) { return IWebSocketResource::Make(); };
   m_sharedState->Module = this;
+  s_sharedState = weak_ptr<SharedState>(m_sharedState);
```

Comment:
>s_sharedState [](http://example.com/codeflow?start=2&length=13)

In the system that has multiple instances it must not work. Two instances will race for this variable and it sounds wrong to me.